index.html

```diff
@@ -177,55 +167,64 @@
         </section>
 
         <section>
-          <h3>Installation and Setup</h3>
+          <h3>Open The Terminal</h3>
+          <h4>On Mac</h4>
+          <p>Type CMD+SPACE<br>In the box type 'Terminal' and hit enter</p>
+          <h4>On Windows</h4>
+          <p>??? I don't know.</p>
```

Comment:
This page on git-scm.com https://git-scm.com/download/win is a Windows EXE file, but I'm not sure what it comes with. The Git Bash looks pretty cool from what I can see on the web page. I'll add that.